Add sync service to plugin

ChihChengLiang opened the pull request:
### What was wrong?

BeaconChainSyncer is not in Beacon plugin 

### How was it fixed?

Add BeaconChainSyncer to Beacon plugin 

### To-Do


#### Cute Animal Picture

![put a cute animal picture link inside the parentheses](https://preview.redd.it/qgfu1nawe6m31.jpg?width=640&crop=smart&auto=webp&s=41cae224cd0c37d5caa299388cc895bec322718c)

hwwhww commented on trinity/protocol/bcc_libp2p/node.py:
```diff
@@ -603,6 +604,7 @@ def _compare_chain_tip_and_finalized_epoch(self,
         await stream.close()
 
     async def say_hello(self, peer_id: ID) -> None:
+        self.logger.info("Say hello")
```
```suggestion
        self.logger.info("Say hello to %s", str(peer_id))
```

ChihChengLiang commented:
Opened another PR #1125 that changes the base branch to master and addresses the PR feedbacks. I also removed the line `await self.say_hello(peer_id)`, the part has already been addressed in #1120 well.
